onnxmltools/convert/sklearn/operator_converters/KNN.py

```diff
@@ -133,12 +141,105 @@ def convert_sklearn_knn(scope, operator, container):
                        [topk_values_name, topk_indices_name], name=scope.get_unique_operator_name('TopK'), k=knn.n_neighbors)
 
     if operator.type == 'SklearnKNeighborsClassifier':
-        raise NotImplementedError
```

Comment:
Need to add another sub-graph or extend the above one to explain what those lines are doing.

Reply:
The sub-graph is already there. I had added it along with KNN Regressor graph. The only part missing is probability calculation graph.

Reply:
The missing part is what I asked. Thanks.

Reply:
Added.